crates/telio-wg/src/wg.rs

```diff
@@ -1445,7 +1458,13 @@ pub mod tests {
 
         // Connects
         peer.time_since_last_handshake = Some(Duration::from_secs(94));
-        peer.rx_bytes = Some(1);
+
+        let _ = task_exec!(&wg.task, async move |s| {
```

Comment:
Again this will very easily hide issues in code, it should be done throug update, (poll of wg adapter)

Reply:
Fixed